src/WebQueryInterceptor.php

```
@@ -20,8 +20,8 @@ public function __construct(
     ) {
     }
 
-    /** @return Pages<mixed>|array<string, mixed> */
-    public function invoke(MethodInvocation $invocation): Pages|array
+    /** @return Pages<mixed>|array<string, mixed>|string */
+    public function invoke(MethodInvocation $invocation): Pages|array|string
```

PHPStan (2.0.3)

Method Ray\MediaQuery\WebQueryInterceptor::invoke() never returns Ray\MediaQuery\Pages<mixed> so it can be removed from the return type.


(return.unusedType)